src/app/atoms/Alert.tsx

```diff
@@ -5,7 +5,7 @@ import classNames from 'clsx';
 import { ReactComponent as CloseIcon } from 'app/icons/close.svg';
 import { t } from 'lib/i18n';
 
-type AlertProps = HTMLAttributes<HTMLDivElement> & {
+type AlertProps = Omit<HTMLAttributes<HTMLDivElement>, 'title'> & {
```

Comment:
why you omit 'title' and after that add it?

Reply:
Because I need separate props